### PR TITLE
WP Multisite Troubleshooting

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes: #5873 

## Summary

**[WordPress Site Networks - Troubleshoot](https://pantheon.io/docs/guides/multisite/debug)** - Documented an issue with cookies for WP admin logins.

Also did some markdown cleanup.

## Post Launch

- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/guides/multisite/05-debug.md [hunk 1]
```diff
@@ -10,36 +10,44 @@ type: guide
...
 
 ## Troubleshooting
 
+### Error: Cookies are blocked or not supported by your browser.
+
+You must enable cookies for WordPress multisite users to log in to their admin interface. Users may need to add this in their `wp-config.php` file:
+
+```php:title=wp-config.php
+define('COOKIE_DOMAIN', $_SERVER['HTTP_HOST'] );
+```
```
Comment (alexfornuto) [inline source/content/guides/multisite/05-debug.md:47]: For technical review, this is the relevant section.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-5936](https://pr-5936--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `74` | `78` | `4` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `93` | `93` | `0` |

Lighthouse Audit Reports:

- [`pr-5936` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23103/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/23103/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-5936--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/guides/multisite/05-debug.md (hunk 1 above)
Comment (jrastaban) [reply source/content/guides/multisite/05-debug.md:47]: I'm not sure why this is under multisite, this occurs on regular blogs as well.

I think the terms here aren't the most clear either. The "You" I think of as a site developer, they don't enable cookies, it is the individual "multisite users" logging in, who need to enable cookies.  "Users" don't have a php file to update, this is a change that the site developer makes.

Lastly, I'm not sure when or why this cookie domain change would be used. I don't have this and things worked fine. 

Is this a second line of trouble shooting if the user has cookies enabled but the site still doesn't work? Does this only work for multisite or could we use this code on a single site fix.

I could see this code as a possible remedy for a challenge that can arise with using custom domains on sub-sites that don't match the source site's domain but I think more context would be needed.

Comment (alexfornuto) [reply source/content/guides/multisite/05-debug.md:47]: CC @panth-jrefuerzo

Comment (carl-alberto) [reply source/content/guides/multisite/05-debug.md:47]: Hi @jrastaban @alexfornuto , we stumbled upon that error while working with a customer on this chat 51746600137739, then we tried the suggestions in here https://wordpress.stackexchange.com/questions/166181/cant-log-in-error-cookies-are-blocked-or-not-supported-by-your-browser-you and it seemed to work. I was helping out @panth-jrefuerzo in doing his first docs issue so asked him to open up this thread. We tried it with the customer that has a multisite but if it also works on non-MU site, would it be a good addition to the troubleshooting section and which part of the docs would this troubleshooting fit to? Thanks!

Comment (alexfornuto) [reply source/content/guides/multisite/05-debug.md:47]: Pinged'd @jrastaban on Slack to take another look.

Comment (alexfornuto) [reply source/content/guides/multisite/05-debug.md:47]: OK, I've added this note as part of our basic troubleshooting doc in progress (commit 7ad2c4959c813f6663c7cf6723aee16bb2ad375e). Would this be acceptable to everyone involved?

**Comment by alexfornuto** [comment]
I'm closing this as resolved by the inclusion of this section in [Basic Troubleshooting](https://pantheon.io/docs/basic-troubleshooting#error-cookies-are-blocked-or-not-supported-by-your-browser). Thanks to all involved!

**Pull request closed without merging**